src/common/cluster-store.ts

```diff
@@ -132,21 +145,30 @@ export class ClusterStore extends BaseStore<ClusterStoreModel> {
   }
 
   @action
-  protected fromStore({ activeCluster, clusters = [] }: ClusterStoreModel = {}) {
+  protected fromStore({ activeCluster, clusters = [], iconOrder = {} }: ClusterStoreModel = {}) {
     const currentClusters = this.clusters.toJS();
     const newClusters = new Map<ClusterId, Cluster>();
     const removedClusters = new Map<ClusterId, Cluster>();
 
+    this.clusterOrders = iconOrder;
+
     // update new clusters
-    clusters.forEach(clusterModel => {
+    for (const clusterModel of clusters) {
```

Comment:
Why `for-of`? I would stick to functional style with `.forEach, .map, etc.` if there is no special case, like `break` for for-loop.

Reply:
Why do we prefer forEach over `for .. of`?

Reply:
because it's more readable / modern / componable, etc. (IMO)

Reply:
I don't understand how in this case it is more readable. From my research `for..of` is faster than `.forEach` and was actually introduced after `.forEach`. As for compoundable, if we are not doing filtering then I would argue they are the same.